docs/linux/helper-function/index.md

```diff
@@ -9,6 +9,15 @@ Helper functions are functions defined by the kernel which can be invoked from e
 
 Helper functions can have a large variety of purposes. This page attempts to categorize them by function.
 
+## Stability guarantees
+
+Although helper functions are not to be used from userspace, but from eBPF programs, they are declared as part of the UAPI (the userspace API): [include/uapi/linux/bpf.h](https://github.com/torvalds/linux/blob/67784a74e258a467225f0e68335df77acd67b7ab/include/uapi/linux/bpf.h#L1828).
+
+As a consequence, helper functions enjoy the well-known stability guarantees of the Linux Kernel (userspace) API.
+Meaning that you can rely on helper functions not disappearing or changing accross kernel versions.
+
+If the stability of the interaction between the eBPF program and the kernel (accross different versions) is a concernt, then helper functions should be preferred over [KFuncs](../index.md#kfuncs).
```

Comment:
Not sure this line makes sense, you will not find helper functions with the same functionality as KFuncs since per kernel maintainer policy new helper functions are not allowed anymore. KFuncs are the way forward